src/components/Filters/FilterTags/FilterTags.js

```diff
@@ -57,13 +57,16 @@ class FilterTags extends Component {
     })),
     removeFilter: PropTypes.func.isRequired,
     onClose: PropTypes.func.isRequired,
-    getSourceLabel: PropTypes.func.isRequired,
   };
 
   static defaultProps = {
     filters: []
   };
 
+  static contextTypes = {
```

Comment:
Connecting a specific selector to FilterTags couples it to specific filters, and we would need to change it every time we need a new selector for a new filter. Using the store from the context and passing it to the filterToLabel will let us just implement that function for the filter outside of this component.